microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -38,13 +40,72 @@ The following code outlines how to declare permissions in the manifest file:
 ]
 ```
 
+#### [Manifest V3](#tab/v3)
+
+```json
+"permissions": [
+  "tabs",
+  "bookmarks",
+  "unlimitedStorage"
+],
+"host_permissions": [
+  "http://www.blogger.com/",
+  "http://*.google.com/"
+]
+```
+
+---
+
+
+<!-- ====================================================================== -->
+
+Here's an example of the permissions part of a manifest file:
+
+#### [Manifest V2](#tab/v2)
```

Comment:
move V3 to be first

Reply:
Got it; will do! Thanks @mikehoffms

Reply:
We merged a new PR into "main" that moved V3 tab to be the left tab, then we merged "main" branch into this PR's branch.  So, the suggested change now appears here, and I can click the Resolve Conversation button here.

Reply:
Updated as recommended.

Reply:
To confirm that this has been updated, see the default Files Changed tab instead of the present Outdate comment:
https://github.com/MicrosoftDocs/edge-developer/pull/2101/files
Find "Here's an example of the permissions part of a manifest file:"
Confirmed that the pink/green lines look ok.   check [a commit-specific preview view](https://github.com/MicrosoftDocs/edge-developer/blob/d0786db4764b2c65f8b36d2b5e63ba7151c93706/microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md):
Looks correct 2x; can assume the tab-sets work.  Underlines (horizontal rules/lines) appear where expected.  Checked Raw listing - the 3 hyphens are where expected.

Resolving comment.